### PR TITLE
feat(events): tracks events

### PR DESCRIPTION
**Pull request by yairans**

### CODE REVIEW

File: src/playkit.js [hunk 1]
```diff
@@ -9,6 +9,10 @@ import Track from './track/track'
 import VideoTrack from './track/video-track'
 import AudioTrack from './track/audio-track'
 import TextTrack from './track/text-track'
+import FakeEventTarget from './event/fake-event-target'
+import FakeEvent from './event/fake-event'
+import {CUSTOM_EVENTS as CustomEvents} from './event/events'
+import * as TestUtils from './utils/test-utils'
```
Comment (OrenMe) [inline src/playkit.js]: Why do we need this for production?

Comment (dan-ziv) [inline src/playkit.js]: Adapters... 
Will be removed if we will use base adapter.

Comment (OrenMe) [inline src/playkit.js]: Please find other solutions for testing, we can't have testing utils leak out to our production.
test utils should be kept under tests, and not under src lib

**Comment by dan-ziv** [comment]
Agree.

File: src/playkit.js (hunk 1 above)
Comment (dan-ziv) [inline src/playkit.js]: 100%

Comment (dan-ziv) [inline src/playkit.js]: done

**Pull request closed without merging**